### PR TITLE
feat: add test case execution related promethus metrics

### PR DESCRIPTION
**Pull request by LinuxSuRen**
> We highly recommend you read [the contributor's documentation](https://github.com/LinuxSuRen/api-testing/blob/master/CONTRIBUTING.md) before starting the review process especially since this is your first contribution to this project.
>
> It was updated on 2024/5/27

**What type of PR is this?**

**What this PR does / why we need it**:
See also

image

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
hi @SamYSF I just added more metrics.

File: pkg/server/remote_server.go [hunk 1]
```diff
@@ -494,8 +496,29 @@ func (s *server) GetTestCase(ctx context.Context, in *TestCaseIdentity) (reply *
...
+	Name: "atest_execution_fail",
+	Help: "The total number of request execution fail",
+})
+
 func (s *server) RunTestCase(ctx context.Context, in *TestCaseIdentity) (result *TestCaseResult, err error) {
 	var targetTestSuite testing.TestSuite
+	ExecutionCountNum.Inc()
+	defer func() {
+		if result.Error == "" {
+			ExecutionFailNum.Inc()
```
Comment (SamYSF) [inline pkg/server/remote_server.go:517]: Why ExecutionFailNum add when error is null?

Comment (LinuxSuRen) [reply pkg/server/remote_server.go:517]: Thanks for pointing.

**Review by SamYSF — APPROVED** [review]

**Review by yuluo-yx — APPROVED** [review]
LGTM.

**Pull request merged**